doc/enterprise-shield/p_enterprise_shield_cluster.md

```diff
@@ -0,0 +1,132 @@
+-   [Home](../../index.md)
+-   [Documentation](../index.md)
+-   Security with KAAZING Gateway
+
+Walkthrough: Configure Enterprise Shield™ for High Availability ![This feature is available in KAAZING Gateway - Enterprise Edition](../images/enterprise-feature.png)
+=================================================================
+
+This step extends the use case to configure an Enterprise Shield™ configuration described in [Walkthrough: Configure Enterprise Shield](p_enterprise_shield_config.md) into a cluster configuration that has two Gateways running in the DMZ and a matching set of Gateways in the trusted network. This configuration provides maximum security and also high availability because it pairs each internal Gateway to one explicit cluster member in the DMZ to provide failover (system resilience) during hardware or network failures. The complete configuration files for the cluster are available on `kaazing.org` at `enterprise-shield-use-case-3-cluster.xml`.
+
+**Note:** Enterprise Shield does not affect the behavior of the cluster in any way. Clients initiate requests in the same way and are unaware that the cluster is configured for reverse connectivity. You connect with a client in the same way as you would for a cluster without Enterprise Shield.
+
+In Figure 1, there are two Gateways running in the DMZ and a matching set of two Gateways on the trusted network. Thus, each internal Gateway is paired to one explicit cluster member in the DMZ.
+
+![Gateway Topology Showing Reverse Connectivity in a Cluster](../images/f-es-maxsecurity-cluster.png)
+
+**Figure 1: Enterprise Shield Topology Configured for High Availability**
+
+The key to making the Enterprise Shield configuration highly available is to configure clustering on each cluster member, and have an equivalent number of cluster members in the DMZ and the trusted network.
+
+The following steps provide a high-level overview about cluster configuration. See [Configure a KAAZING Gateway Cluster](../high-availability/p_high_availability_cluster.md) for detailed information about cluster configuration and the [cluster](../admin-reference/r_configure_gateway_cluster.md) element.
+
+-   Update the properties element to use a numbered value for the DMZ backplane variable---such as `gateway1`---to provide a unique name for the Gateway cluster pair:
```

Comment:
> ... to use a numbered value for the DMZ backplane variable ...

This sounds like we are telling them to use a numbered value. That is not necessary. What is necessary is that each hostname be unique, and we chose numbering as an easy way to do that.

Readers may think they must use numbered based on this description. And it's possible their hostnames will not have numbering, and they'll make the hostnames unique in some other way. Therefore this instruction may confuse them.